### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ochiai-kfj**
first test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # test2
 # test2
+
+President Joe Biden heads into the holiday weekend facing a series of planned events that could have serious consequences as his fate as the presumptive Democratic presidential nominee hangs in the balance.
+
```
Comment (yu2scalar) [inline README.md:5]: Is it fake?

**Review by yu2scalar — CHANGES_REQUESTED** [review]
test

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # test2
 # test2
+
+President Joe Biden heads into the holiday weekend facing a series of planned events that could have serious consequences as his fate as the presumptive Democratic presidential nominee hangs in the balance.
```
Comment (yu2scalar) [inline README.md]:
```suggestion
President Joe Fak Biden heads into the holiday weekend facing a series of planned events that could have serious consequences as his fate as the presumptive Democratic presidential nominee hangs in the balance.
```

**Review by yu2scalar — CHANGES_REQUESTED** [review]
change

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
 # test2
 # test2
+
+President Joe Fak Biden heads into the holiday weekend facing a series of planned events that could have serious consequences as his fate as the presumptive Democratic presidential nominee hangs in the balance.
```
Comment (yu2scalar) [inline README.md:4]:
```suggestion
President Joe Fake Biden heads into the holiday weekend facing a series of planned events that could have serious consequences as his fate as the presumptive Democratic presidential nominee hangs in the balance.
```

**Review by yu2scalar — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,6 @@
...
+
+First on the list is a closed-door crisis meeting tonight at the White House with several Democratic governors, including California Gov. Gavin Newsom, Michigan Gov. Gretchen Whitmer and Illinois Gov. J.B. Pritzker – all names being bandied about as potential replacements.
```
Comment (yu2scalar) [inline README.md:6]:
```suggestion
First on the list is a closed-door crisis meeting tonight at the White House with several Democratic governors, including California Gov. Gavin Newsom, Michigan Gov. Gretchen Whitmer and Illinois Gov. J.B. Pritzker – all names being bandied about as potential replacements.
Add more changes
```

Any comments?

**Review by yu2scalar — APPROVED** [review]
Approve test

**Comment by ochiai-kfj** [comment]
Merged for the first test

**Pull request merged**